docs-source/spring/content/setup/_index.md

```diff
@@ -125,3 +127,34 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
     <!-- spellchecker-disable -->
     {{< img name="oci-stack-apply-logs" size="large" lazy=false >}}
     <!-- spellchecker-enable -->
+
+## Setup the local workstation
+
+1. Setting Up Cluster Access
+
+    To access a cluster using kubectl installed (see the [Kubernetes access](./cluster-access)) locally. If you haven't already done so,
+
+    - install kubectl (see the [kubectl documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/)).
+    - generate an API signing key pair
+        If you already have an API signing key pair, go straight to the next step. If not:
+
+        1. Use OpenSSL commands to generate the key pair in the required PEM format. If you're using Windows, you'll need to install Git Bash for Windows and run the commands with that tool. See [How to Generate an API Signing Key](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#two).
+        2. Copy the contents of the public key to the clipboard (you'll need to paste the value into the Console later).
+
+    - add the public key value of the API signing key pair to the User Settings for your username
+       1. In the top-right corner of the Oracle Cloud Infrastructure Console, open the **Profile** menu (User menu icon) and then click **User Settings** to view the details.
+       2. Click **Add Public Key**.
+       3. Paste the public key's value into the window and click **Add**.
+
+          The key is uploaded and its fingerprint is displayed (for example, d1:b2:32:53:d3:5f:cf:68:2d:6f:8b:5f:77:8f:07:13).
+
+    - install and configure the Oracle Cloud Infrastructure CLI
+        1. Install the Oracle Cloud Infrastructure CLI version 2.6.4 (or later). See [Quickstart](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm#Quickstart).
+        2. Configure the Oracle Cloud Infrastructure CLI. See [Configuring the CLI](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Configuring_the_CLI).
+
+
+2. Install Oracle Backend as a Service for Spring Cloud command-line
+
+    The **Oracle Backend as a Service for Spring Cloud** command-line tool, `oracle-spring`, is available for Linux and Mac systems. Download the binary you want from the [Releases](https://github.com/oracle/microservices-datadriven/releases/tag/OBAAS-1.0.0) page and add it to your PATH environment variable.  You may like to rename the binary to remove the suffix.
+
+    If you're environment is a Linux or Mac machine you need to execute `chmod +x` on the downloaded binary. Also if your environment is a Mac you need execute the following command `sudo xattr -r -d com.apple.quarantine <downloaded-file>` otherwise will you get a security warning and the CLI will not work.
```

Comment:
you need to rebase, this is already updated

Reply:
this is coming from another PR not merged yet.